examples/stable_diffusion_v2/configs/v2-train.yaml

```diff
@@ -33,6 +33,7 @@ model:
         #num_heads: 8  #wukong
         num_head_channels: 64  # SD_VERSION v2.0
         use_spatial_transformer: True
+        enable_flash_attention: True
```

Comment:
what about other configs, v1, inference, etc.

Reply:
Only this config file is currently verified